src/main/resources/lang/default.lang

```diff
@@ -2275,6 +2275,18 @@ transform reasons:
 	unknown: unknown
 	infection: infection, villager infection
 
+# -- Teleport Flags --
+teleport flags:
+	retain_open_inventory: opened inventory, open inventory, inventory
+	retain_passengers: passengers, passenger
+	retain_vehicle: vehicle
+	retain_direction: direction
+	retain_pitch: pitch
+	retain_yaw: yaw
+	retain_x: x coordinate, x-coordinate
+	retain_y: y coordinate, y-coordinate
+	retain_z: z coordinate, z-coordinate
```

Comment:
These are for retaining the relative velocity on the respective axes, so i'd like to see the name reflect that instead. 
Ex: `directional velocity, view momentum, x-coordinate movement, y velocity` etc.

It'd also be nice to have a flag that does XYZ all at once, like how retain_direction works